openmdao/docs/feature_reference/building_blocks/solvers/backtracking/index.rst

```diff
@@ -4,10 +4,18 @@
 Linesearch/Backtracking
 ***********************
 
+Backtracking line searches are subsolvers that can be specified in the `line_search` attribute
+of a NewtonSolver and are used to pull back to a reasonable point when a Newton step goes to far. This
+can occur when a step causes output variables to exceed their specified lower and upper bounds. It can
+also happen in more complicated problems where a full Newton step happens to take you well past the nonlinear solution,
+even to an area where the residual norm is worse than the initial point. Specifying a line_search can
```

Comment:
suggest "line search" or alternatively "a value for `line_search`" rather than mixing English and code

Reply:
Fixed